src/Classes/Models/HiringPartnerModel.php

```diff
@@ -56,6 +56,17 @@ public function getCompanySize() :array
         return $query->fetchAll();
     }
 
+    /** Method does a query request that picks up the id and name to be displayed on the dropdown
+     *
+     * @return array
+     */
+    public function getCompanyName() :array
+    {
+        $query = $this->db->prepare("SELECT `id`,`name` FROM `hiring_partner_companies`");
+        $query->execute();
+        return $query->fetchAll();
+    }
+
```

Comment:
Can you give this to Liz so it can be part of her task and remove it from this branch?

Reply:
sure